binutils build fails when makeinfo is missing

```
WARNING: `makeinfo' is missing on your system.  You should only need it if
         you modified a `.texi' or `.texinfo' file, or any other file
         indirectly affecting the aspect of the manual.  The spurious
         call might also be the consequence of using a buggy `make' (AIX,
         DU, IRIX).  You might want to install the `Texinfo' package or
         the `GNU make' package.  Grab either from any GNU archive site.
make[3]: *** [bfd.info] Error 1
make[3]: Leaving directory `/media/3T/git-mirror/musl-cross/binutils-2.23.2/build1/bfd/doc'
```

--- a/build.sh
+++ b/build.sh
@@ -47,6 +47,9 @@ then
 else
     fetchextract http://ftp.gnu.org/gnu/binutils/ binutils-$BINUTILS_VERSION .tar.bz2
 fi
+
+sed -i 's,MAKEINFO="$MISSING makeinfo",MAKEINFO=true,g' \
+    binutils-$BINUTILS_VERSION/configure
 buildinstall 1 binutils-$BINUTILS_VERSION --target=$TRIPLE --disable-werror \
     --with-sysroot="$PREFIX"/"$TRIPLE" \
     $BINUTILS_CONFFLAGS

